samplers/parser.go

```diff
@@ -14,6 +14,15 @@ import (
 	"github.com/stripe/veneur/ssf"
 )
 
+const DogStatsDCheckIdentifierKey = "dogstatsd_sc"
+const DogStatsDCheckHostnameTagKey = "dogstatsd_hostname"
+const DogStatsDEventAggregationKeyTagKey = "dogstatsd_ak"
+const DogStatsDEventAlertTypeTagKey = "dogstatsd_at"
+const DogStatsDEventHostnameTagKey = "dogstatsd_hostname"
+const DogStatsDEventIdentifierKey = "dogstatsd_ev"
+const DogStatsDEventPriorityTagKey = "dogstatsd_pri"
+const DogStatsDEventSourceTypeTagKey = "dogstatsd_st"
```

Comment:
Hrm, is `samplers` the right place for this to live? Are we only ever going to use these tags internally?

Reply:
(If it would be used externally, maybe we should put it in `ssf` or `protocol` even; something that clients can use that doesn't pull in like 60% of veneur)